br/pkg/restore/data.go

```diff
@@ -57,7 +57,7 @@ func RecoverData(ctx context.Context, resolveTS uint64, allStores []*metapb.Stor
 		return totalRegions, errors.Trace(err)
 	}
 
-	if err := recovery.PrepareFlashbackToVersion(ctx); err != nil {
+	if err := recovery.PrepareFlashbackToVersion(ctx, resolveTS, restoreTS-1); err != nil {
```

Comment:
Can we guarantee that no other write on `restoreTS-1`? Otherwise, it may overwrite the old value.

/cc @fengou1

Reply:
Yes, In EBS restore. workflow as follows
1. start pd
2. set pd/tikv recoverymode
3. start tikv
4. prepare/flashback version
5. start tidb

there is no chance to write any data before step 5 (start tidb)